js/main.js

```diff
@@ -1,2 +1,94 @@
 $(function () {
+
+  const botChoices = ["rock", "paper", "scissors"]
+  let botScore = 0
+  let userScore = 0
+
+  $("#rock").click(() => {
+    console.log("Rock clicked")
+    let botChoice = generate()
+    compare("rock", botChoice)
```

Comment:
Very nice! I like that you separated concerns here and passed information to other functions based on a click.